src/client/utils/renderStyleguide.tsx

```diff
@@ -45,6 +47,13 @@ export default function renderStyleguide(
 	return (
 		<StyleGuide
 			codeRevision={codeRevision}
+			// only caclulate css revisions in dev when hot is on to avoid
+			// stringifying the styles in production
+			cssRevision={
+				module.hot
+					? hashSum({ theme: styleguide.config.theme, styles: styleguide.config.styles })
```

Comment:
We're already destructuring config above, so we can add `theme` and `styles` there too.

```suggestion
					? hashSum({ theme, styles })
```